docs/changelog.md

```diff
@@ -5,8 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Prepare and run action
 
-## [0.1.0]
+## [v0.1.0]
```

Comment:
Is CI release workflow will work with such style? We make a release, but I don't remember detailes was there any problems with version naming and release notes

Reply:
We had promlems with it because script didn't find `v0.1.0`